web/src/app/components/login-page/login-page.component.ts

```diff
@@ -47,20 +48,17 @@ export class LoginPageComponent implements OnInit {
       });
   }
 
-  saveUser(user, provider) {
+  saveUser(user: User, provider: string) {
     this.userService.saveUser(user)
       .subscribe((userData) => {
         this.saveUserSocialDetails(userData, provider);
       });
   }
 
-  saveUserSocialDetails(userData, provider) {
+  saveUserSocialDetails(userData: object, provider: string) {
```

Comment:
Can we use a more specific type, rather than `object`? Object is almost like `any`

Reply:
Sure @eddiejaoude . Will change it.